samples/local_attestation/common/crypto.cpp

```diff
@@ -173,7 +174,7 @@ bool Crypto::Encrypt(
 
     // Encrypt the data.
     res = mbedtls_rsa_pkcs1_encrypt(
-        mbedtls_pk_rsa(key),
+        rsa_context,
```

Comment:
Nice catch, might also want to replace the usage on line 190 for consistency.

Reply:
I take no credit for catching this, this was a case where one sample did it one way and the other sample did it the other way.  So someone else previously caught it but only fixed one sample :)

Reply:
> File remoteattestation_sample_details.png2 was added but I don't see it being referenced?

Not intentional, thanks for catching, removed.